src/dodal/plans/load_eiger_metadata_plan.py

```diff
@@ -0,0 +1,144 @@
+import time
+
+import bluesky.plan_stubs as bps
+from ophyd_async.core import DetectorTrigger
+from ophyd_async.fastcs.eiger import EigerDetector, EigerTriggerInfo
+
+from dodal.devices.detector import DetectorParams
+from dodal.log import LOGGER
+
+
+def load_metadata(
+    eiger: EigerDetector,
+    enable: bool,
+    detector_params: DetectorParams,
+):
+    start = time.time()
+    assert detector_params.expected_energy_ev
+    yield from bps.abs_set(eiger.odin.capture, 0)
+    LOGGER.info(f"Stopping Odin: {time.time() - start}s")
+    yield from set_cam_pvs(eiger, detector_params, wait=True)
+    LOGGER.info(f"Setting CAM PVs: {time.time() - start}s")
+    yield from set_odin_pvs(eiger, detector_params, wait=True)
+    LOGGER.info(f"Setting Odin PVs: {time.time() - start}s")
+    yield from change_roi_mode(eiger, enable, detector_params, wait=True)
+    LOGGER.info(f"Changing ROI Mode: {time.time() - start}s")
+    yield from bps.abs_set(eiger.odin.num_frames_chunks, 1)
+    LOGGER.info(f"Setting # of Frame Chunks: {time.time() - start}s")
+    yield from set_mx_settings_pvs(eiger, detector_params, wait=True)
```

Comment:
For next time we're testing we should try removing these individual waits, I think the arm should all still work